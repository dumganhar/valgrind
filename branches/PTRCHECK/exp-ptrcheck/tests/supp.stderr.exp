
Warning: invalid file descriptor -1 in syscall write()

ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 3 from 3)
