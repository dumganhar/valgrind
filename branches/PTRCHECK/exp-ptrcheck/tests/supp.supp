{
   <insert a suppression name here>
   Annelid:LoadStore
   fun:main
   fun:__libc_start_main
   obj:*/annelid/tests/supp
}
{
   <insert a suppression name here>
   Annelid:Arith
   fun:main
   fun:__libc_start_main
   obj:*/annelid/tests/supp
}
{
   <insert a suppression name here>
   Annelid:SysParam
   write(buf)
   fun:__GI___libc_write
   fun:__libc_start_main
   obj:*/annelid/tests/supp
}
