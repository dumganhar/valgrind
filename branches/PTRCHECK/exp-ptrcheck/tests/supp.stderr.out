
Invalid read of size 1
   at 0x........: main (supp.c:13)
 Address 0x........ is 1 bytes before the accessing pointer's
 legitimate range, a block of size 6 alloc'd
   at 0x........: malloc (vg_replace_malloc.c:...)
   by 0x........: main (supp.c:12)

Invalid arguments to Add32/Add64
   at 0x........: main (supp.c:14)
 Both args derived from address 0x........ of 6-byte block alloc'd
   at 0x........: malloc (vg_replace_malloc.c:...)
   by 0x........: main (supp.c:12)
Warning: invalid file descriptor -1 in syscall write()

ERROR SUMMARY: 2 errors from 2 contexts (suppressed: 0 from 0)
